ckanext/smdh/templates/user/list.html

```diff
@@ -0,0 +1,23 @@
+{% ckan_extends %}
+
+{% block users_list %}
+  {% if h.isAdmin() %}  
+    <ul class="user-list">
+      {% block users_list_inner %}
+        {% for user in page.items %}
+          <li>{{ h.linked_user(user['name'], maxlength=20) }}</li>
+          
+        {% endfor %}
+      {% endblock %}
+    </ul>
+  {% else %}
+    <p>Sorry you do not have permissions to view all users</p>
```

Comment:
Should we wrap this <p> in a div with class "alert alert-error" to make it stand out a bit more?

Reply:
After looking at it, looks a lot better. Didn't know about that class. commit made